react/components/SearchBar/components/SearchBar.js

```diff
@@ -97,6 +112,24 @@ const SearchBar = ({
     [navigate, attemptPageTypeSearch, customSearchPageUrl]
   )
 
+  const validateInput = () => {
+    if (minSearchTermLength && inputValue.length < minSearchTermLength) {
+      return intl.formatMessage({
+        id: 'store/search.searchTermTooShort',
+      })
+    }
+
+    return undefined
```

Comment:
Maybe this could be better
```suggestion
    return null
```